example/provider-extensions/seed/configure-discovery-server.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+#
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+usage() {
+  echo "Usage:"
+  echo "> configure-discovery-server.sh [ -h | <garden-kubeconfig> <seed-kubeconfig> ]"
+  echo
+  echo ">> For example: configure-discovery-server.sh ~/.kube/garden-kubeconfig.yaml ~/.kube/seed-kubeconfig.yaml"
+
+  exit 0
+}
+
+if [ "$1" == "-h" ] || [ "$#" -ne 2 ]; then
+  usage
+fi
+
+garden_kubeconfig=$1
+seed_kubeconfig=$2
+
+use_shoot_info="false"
+temp_shoot_info=$(mktemp)
+cleanup-shoot-info() {
+  rm -f "$temp_shoot_info"
+}
+trap cleanup-shoot-info EXIT
+
+ensure-gardener-dns-annotations() {
+  local namespace=$1
+  local name=$2
+  local domain=$3
+
+  kubectl annotate --overwrite --kubeconfig "$seed_kubeconfig" svc -n "$namespace" "$name" \
+    dns.gardener.cloud/class=garden \
+    dns.gardener.cloud/ttl="60" \
+    dns.gardener.cloud/dnsnames="$domain" \
+    cert.gardener.cloud/commonname="$domain" \
+    cert.gardener.cloud/dnsnames="$domain" \
+    cert.gardener.cloud/purpose=managed \
+    cert.gardener.cloud/secretname=tls
+}
+
+gardener_issuer_domain=
+
+if kubectl get configmaps -n kube-system shoot-info --kubeconfig "$seed_kubeconfig" -o yaml > "$temp_shoot_info"; then
+  use_shoot_info="true"
+  echo "Getting config from shoot"
+  gardener_issuer_domain=issuer.$(yq -e '.data.domain' "$temp_shoot_info")
```

Comment:
I guess the part putting wrapping this issuer into `https://<issuer>/garden/workload-identity/issuer` and putting it into the file `example/provider-extensions/garden/controlplane/workload-identity-issuer.yaml` is still missing?

Reply:
I managed to bring such local environment and indeed the GAPI is not re-configured to use the correct issuer.

Reply:
I left this as a manual step but it makes sense to automate it if possible. I will address the comment in a following commit